cmd/gapit/export_replay.go

```diff
@@ -98,5 +114,155 @@ func (verb *exportReplayVerb) Run(ctx context.Context, flags flag.FlagSet) error
 	if err := client.ExportReplay(ctx, capturePath, device, verb.Out, opts); err != nil {
 		return log.Err(ctx, err, "Failed to export replay")
 	}
+
+	if verb.Apk != "" {
+		// Create stand-alone APK
+		replayAPK := filepath.Base(verb.Apk)
+		replayPackage := strings.TrimSuffix(replayAPK, ".apk")
+		log.I(ctx, "Create replay apk: %s with package name %s", replayAPK, replayPackage)
+
+		boxedCapture, err := client.Get(ctx, capturePath.Path(), nil)
+		if err != nil {
+			return log.Err(ctx, err, "Failed to load the capture")
+		}
+		capture := boxedCapture.(*service.Capture)
+
+		refAPKPath, err := layout.GapidApk(ctx, capture.ABI)
+		if err != nil {
+			return err
+		}
+
+		// Extract reference APK
+		if err := execCommand(ctx, "unzip", "-o", refAPKPath.String()); err != nil {
```

Comment:
I will re-add these comments in case they got buried from the previous MRs.

We should avoid using command-line executables that we cannot guarantee exist. zip/unzip are not present on windows for example.